index.html

```diff
@@ -161,7 +165,7 @@
 <!-- - - - - - - - - - - - - - -  Introduction  - - - - - - - - - - - - - - -->
 <section class="informative"> <h2>Introduction</h2>
   <div>
-    NFC user scenarios can be grouped as follows:
+    Web NFC user scenarios can be grouped as follows:
```

Comment:
In the Introduction section it would not be out of place to enumerate a few NFC use cases, while clarifying which ones are currently implemented by Web NFC. The classification was meant to provide generic information. Once we remove the generic use cases, I don't know if it makes sense to put the use cases here, and have a full Use Cases section as well (with basically the same content). Probably doesn't hurt.